molecule/default/tests/test_create_bootable_volume.py

```diff
@@ -0,0 +1,47 @@
+import utils
+import os
+import pytest
+import testinfra.utils.ansible_runner
+
+testinfra_hosts = testinfra.utils.ansible_runner.AnsibleRunner(
+    os.environ['MOLECULE_INVENTORY_FILE']).get_hosts('nova_compute')[:1]
+
+
+@pytest.mark.test_id('3c469966-4fcb-11e8-a604-6a0003552100')
+@pytest.mark.jira('asc-258')
+def test_create_bootalbe_volume(host):
+    """Test to verify that a bootable volume can be created based on a Glance image
+
+    Args:
+        host(testinfra.host.Host): A hostname in dynamic_inventory.json/molecule.yml
+    """
+
+    null = None
+    false = False
+    volume_name = 'test_volume'
+
+    image_id = utils.get_image_id("Cirros-0.3.5", host)
```

Comment:
The hard coded cirros version seems brittle to me. Can this at least be set to a variable so that it is easier to spot and update when this version is updated in the future?